homeassistant/components/zwave/helper.py

```diff
@@ -0,0 +1,11 @@
+"""Helpers for zwave specific config validation using voluptuous."""
+import re
+import voluptuous as vol
+
+def zwave_network_key(value):
+    """Validate a 16 byte value for zwave network keys."""
+    regex = re.compile(r'(0x\w\w,\s?){15}0x\w\w')
```

Comment:
You don't need this. Just use `vol.Match` https://github.com/alecthomas/voluptuous/blob/master/voluptuous/validators.py#L295

Reply:
It is great when the actual solution can be boiled down to a single line of code.